tests/foreman/ui/test_registration.py

```diff
@@ -0,0 +1,73 @@
+"""Tests for registration.
+
+:Requirement: Registration
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Registration
+
+:CaseAutomation: Automated
+
+:CaseImportance: Critical
+
+:Team: Rocket
+
+:TestType: Functional
+
+:Upstream: No
+"""
+
+import pytest
+
+
+@pytest.mark.tier2
+def test_positive_org_loc_change_for_global_registration(
+    session,
+    module_activation_key,
+    module_org,
+    module_location,
+    default_os,
+    target_sat,
+):
+    """Changing the organization and location to check if correct org and loc is updated on the global registration command
+
+    :id: e83ed6bc-ceae-4021-87fe-3ecde1cbf347
+
+    :expectedresults: organization and location is updated correctly on the global registration command
+
+    :CaseImportance: Medium
+    """
+    new_org = target_sat.api.Organization().create()
+    new_loc = target_sat.api.Location().create()
+    new_activation_key = target_sat.api.ActivationKey(organization=new_org).create()
+
+    with session:
+        cmd = session.host.get_register_command(
+            {
+                'general.operating_system': default_os.title,
+                'general.activation_keys': module_activation_key.name,
+                'general.insecure': True,
+            }
+        )
+        expected_pairs = [
+            f'organization_id={module_org.id}',
+            f'location_id={module_location.id}',
+        ]
+        for pair in expected_pairs:
+            assert pair in cmd
+        # changing the org and loc to check if correct org and loc is updated on the registration command
+        session.organization.select(org_name=new_org.name)
+        session.location.select(loc_name=new_loc.name)
+        cmd = session.host.get_register_command(
+            {
+                'general.operating_system': default_os.title,
+                'general.activation_keys': new_activation_key.name,
+                'general.insecure': True,
```

Comment:
We can remove general.operating_system and general.insecure are optional, so it can be removed.
and general.activation_keys will be autoselected as its single AK in org
Could you check and update this PR as per https://github.com/SatelliteQE/airgun/pull/1087 as discussed?

Reply:
Updated